packages/credential-provider-imds/package.json

```diff
@@ -29,6 +29,7 @@
     "@types/jest": "^26.0.4",
     "@types/node": "^10.0.0",
     "jest": "^26.1.0",
+    "nock": "^13.0.2",
```

Comment:
I prefer adding the dev dependency to the project root, just like all the E2E test dependencies. Doing so can reduce the number dependencies user needs to download.